recipes/mysql-connector-cpp/all/conanfile.py

```diff
@@ -0,0 +1,116 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake
+from conan.tools.files import get
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd, cross_building, stdcpp_library
+
+class MysqlConnectorCPPRecipe(ConanFile):
+    name = "mysql-connector-cpp"
+    license = "GPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://dev.mysql.com/doc/connector-cpp/8.0"
+    description = "A Conan package for MySQL Connector/C++ with OpenSSL, Boost, and libmysqlclient"
+    topics = ("mysql", "connector", "cpp", "openssl", "boost", "libmysqlclient", "jdbc", "static")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    @property
+    def _min_cppstd(self):
+        return "17"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "16",
+            "msvc": "192",
+            "gcc": "9" if Version(self.version) >= "8.0.27" else "5.3",
+            "clang": "6",
+        }
+
+    def requirements(self):
+        self.requires("boost/1.82.0")
+        # build faiils on mac with openssl 3x
```

Comment:
```suggestion
        # build faiils on mac with openssl 3x
```
I would suggest adding some extra information about that error, like error message provided by the compiler, so someone could check and fix it, or even, report to the upstream.